ports/esp32/modesp32.c

```diff
@@ -178,6 +178,28 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_0(esp32_hall_sensor_obj, esp32_hall_sensor);
 
 #endif
 
+#if CONFIG_IDF_TARGET_ESP32S3 
+
+#include "driver/temp_sensor.h"
+
+STATIC mp_obj_t esp32_raw_temperature(void) {
+
+    temp_sensor_config_t temp_sensor = {
+            .dac_offset = TSENS_DAC_L2,
+            .clk_div = 6,
+    };
```

Comment:
Why not use `TSENS_CONFIG_DEFAULT()`?  And `TSENS_DAC_DEFAULT` for `.dac_offset`?